pkg/agent/sysadvisor/plugin/qosaware/reporter/headroom_reporter.go

```diff
@@ -69,7 +64,8 @@ func NewHeadroomReporter(emitter metrics.MetricEmitter, metaServer *metaserver.M
 	return &headroomReporterImpl{plugin}, nil
 }
 
-func (r *headroomReporterImpl) Run(ctx context.Context) {
+func (r *headroomReporterImpl) Run(ctx context.Context, wg *sync.WaitGroup) {
+	defer wg.Done()
```

Comment:
why not move all `defer wg.Done()` to qos_aware plugin `Run()`?

Reply:
done